assets/src/pages/room/hooks/useMembraneClient.tsx

```diff
@@ -68,83 +48,84 @@ export function useMembraneClient(
     const webrtc = new MembraneWebRTC({
       callbacks: {
         onSendMediaEvent: (mediaEvent: SerializedMediaEvent) => {
-          console.log({ name: "onSendMediaEvent", mediaEvent: JSON.parse(mediaEvent) });
           webrtcChannel.push("mediaEvent", { data: mediaEvent });
         },
         onConnectionError: (message: string) => {
-          console.log({ name: "onConnectionError", message });
           setErrorMessage("Something went wrong :(");
         },
         // todo [Peer] -> Peer[] ???
```

Comment:
PR for this change:
https://github.com/membraneframework/membrane-webrtc-js/pull/14